be/src/storage/schema_change.cpp

```diff
@@ -269,6 +270,133 @@ Status LinkedSchemaChange::process(TabletReader* reader, RowsetWriter* new_rowse
     return Status::OK();
 }
 
+Status LinkedSchemaChange::convert_from_base_rowset_by_cols(const Tablet* new_tablet, const Tablet* base_tablet,
+                                                            const RowsetSharedPtr src_rowset, RowsetId rid,
```

Comment:
const TabletPtr& instead of pointer. pointer means to change the content.

Reply:
I think we should keep const Tablet* because the function call convert_from_base_rowset_by_cols is just raw point Tablet* but not share pointer

Reply:
The shared_point is the same usage with the naked pointer.